### PR TITLE
init blob detectors - disabled by default, need explicit activation

### PR DESCRIPTION
**Pull request by nicolasvion**
- adding blob storage detectors
- disabled by default, need explicit activation because of some stability pb
- customers' request

### CODE REVIEW

File: modules/integration_azure-storage-account-blob/conf/01-requests-rate-status.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+disabled: true
+signals:
+  rate_success:
+    metric: "Transactions"
+    filter: "filter('responsetype', 'Success')"
+    rollup: rate
+  rate_failed:
+    metric: "Transactions"
+    rollup: rate
+    filter: "not filter('responsetype', 'Success')"
```
Comment (Shr3ps) [inline modules/integration_azure-storage-account-blob/conf/01-requests-rate-status.yaml:16]: hum, filter on Failed?

File: modules/integration_azure-storage-account-blob/conf/02-latency-e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+signals:
+  latency:
+    metric: "SuccessE2ELatency"
+    rollup: average
+  signal:
+    formula:
+      latency.scale(0.001) # Scale to s unit
+rules:
+  critical:
+    threshold: 2
```
Comment (Shr3ps) [inline modules/integration_azure-storage-account-blob/conf/02-latency-e2e.yaml:17]: 2s of latency by default? don't you think it's low by default?

but not very important, we can override this value with the terraform variable.

File: modules/integration_azure-storage-account-blob/conf/readme.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+documentations:
```
Comment (Shr3ps) [inline modules/integration_azure-storage-account-blob/conf/readme.yaml:1]: A real link here would be great :)

**Review by Shr3ps — CHANGES_REQUESTED** [review]

File: modules/integration_azure-storage-account-blob/common-modules.tf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+module "filter-tags" {
```
Comment (xp-1000) [inline modules/integration_azure-storage-account-blob/common-modules.tf:1]: should be named `modules.tf`

File: modules/integration_azure-storage-account-blob/conf/readme.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+  - name: Azure Monitor metrics
+    url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
+source_doc: >-
+  We are using metrics from the
+  [Microsoft.Storage/storageAccounts](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftclassicstoragestorageaccountsblobservices) namespace.
```
Comment (xp-1000) [inline modules/integration_azure-storage-account-blob/conf/readme.yaml:6]: I would add a note about disabled detectors (why disabled by default and when to use them)

**Review by xp-1000 — CHANGES_REQUESTED** [review]
just to minor comments

**Review by xp-1000 — APPROVED** [review]
LGTM thanks

**Review by Shr3ps — APPROVED** [review]

**Pull request merged**